WORKSPACE

```diff
@@ -106,3 +106,13 @@ new_http_archive(
     type = "zip",
     url = "https://github.com/nunit/nunitv2/releases/download/2.6.4/NUnit-2.6.4.zip",
 )
+
+maven_jar(
+    name = "scala-xml",
+    artifact = "org.scala-lang.modules:scala-xml_2.11:1.0.5",
```

Comment:
I have this:
https://bazel-review.googlesource.com/#/c/2410/1

which pulls scala 2.11 and gets the scalac and scalalib there. It would be nice to use the jar provided there once both this and that are merged.